stylesheets/commons/Miscellaneous.less

```diff
@@ -584,6 +584,10 @@ Author(s): FO-nTTaX, Rapture
 	margin: 10px 0;
 }
 
+.spellcard-wrapper hr {
+	height: 0.125rem;
+}
+
```

Comment:
You can nest the `hr` inside the `.spellcard-wrapper` class, so it would look like:
```
.spellcard-wrapper {
  <existing spellcard-wrapper css>
  
  .hr {
    height: 0.125rem;
  }
}
```

Reply:
Thank you for the suggestion. I'll do this from now on!